accessibility-oscilloscope/overlay/usr/bin/wacom-xy

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+DEVICE=$(libinput list-devices | grep "Wacom Intuos BT M Pen" -A1 | awk '/Kernel:/ {print $2}')
```

Comment:
compared to something like the i2c bus for haptics, the kernel can order these input events effectively randomly.
here munge the result of `libinput list-devices` to get the right `/dev/input/event`.